deduplication/src/main/java/gov/cdc/nbs/deduplication/merge/PatientMergeController.java

```diff
@@ -41,22 +40,34 @@ public PatientMergeController(
 
   @GetMapping
   public MatchesRequireReviewResponse getPotentialMatches(
-          @RequestParam(defaultValue = "0", name = "page") int page,
-          @RequestParam(defaultValue = "5", name = "size") int size,
-          @RequestParam(defaultValue = DEFAULT_SORT, name = "sort") String sort) {
+      @RequestParam(defaultValue = "0", name = "page") int page,
+      @RequestParam(defaultValue = "5", name = "size") int size,
+      @RequestParam(defaultValue = DEFAULT_SORT, name = "sort") String sort) {
     return matchesRequiringReviewResolver.resolve(page, size, sort);
   }
 
   @GetMapping("/{patientId}")
   public ResponseEntity<List<PersonMergeData>> getPotentialMatchesDetails(
-          @PathVariable("patientId") Long patientId) {
+      @PathVariable("patientId") Long patientId) {
     return ResponseEntity.ok(mergeGroupHandler.getPotentialMatchesDetails(patientId));
   }
 
-  @PostMapping("/group-no-merge")
-  public ResponseEntity<String> updateGroupNoMerge(@RequestBody GroupNoMergeRequest request) {
+
+  @PostMapping("/unmerge-all/{personId}") // Keep Separate
+  public ResponseEntity<String> unMergeAll(@PathVariable("personId") Long personId) {
```

Comment:
Issue: REST usually doesn't return just a `String` and isn't really necessary here. Can we update this to be `ResponseEntity<Void>` or just `void`. The status code of 200 should be sufficient to alert the user that the request was successfully processed.

If we do want to return some text, then a response object should be created such as 
`public record UnMergeResponse(String message) {}` and then an instance of that be returned.